### ISSUE TITLE
CSS styles not showing right away and outside scripts and stylesheet come up as 404

### ISSUE THREAD

**Issue by jbbow** [eddiemachado/bones#807]
Hi, I am using your bones theme for the very first time. I am having an incredibly hard time with new css (I'm only using css and not LESS or SASS) showing immediately when refreshing, I have to wait 5 minutes or longer for every change.  Obviously i can't work like that, so I'd like to know if I can do something to speed that up.

Also, I am trying to register a new css stylesheet and .js files but everytime I go to the site, it says file not found.  I am at my wits end (regarding myself, not the theme - obviously I don't know enough to be using bones but I can't turn back now).  The site is not live. I can give you a username and password.  

I added 2 scripts files and one css file to the bones.php file and they reside in a folder called, HoverAlls.  I don't even know if I did it right, but I also put a function in the script.js (see below) file which it states its not a function.  This may be way over my head, you can tell me if it is.  :)

```	
// modernizr (without media query polyfill)
		wp_register_script( 'bones-modernizr', get_stylesheet_directory_uri() . '/library/js/libs/modernizr.custom.min.js', array(), '2.5.3', false );

		// register main stylesheet
		wp_register_style( 'bones-stylesheet', get_stylesheet_directory_uri() . '/library/css/style.css', array(), '', 'all' );
		wp_register_style( 'stylesheet' , get_stylesheet_directory_uri() . '/HoverAlls/HoverAlls-2.6.min.css', array(), '', 'all' );

		// ie-only style sheet
		wp_register_style( 'bones-ie-only', get_stylesheet_directory_uri() . '/library/css/ie.css', array(), '' );
		
		
    // comment reply script for threaded comments
    if ( is_singular() AND comments_open() AND (get_option('thread_comments') == 1)) {
		  wp_enqueue_script( 'comment-reply' );
    }

		//adding scripts file in the footer
		wp_register_script( 'bones-js', get_stylesheet_directory_uri() . '/library/js/scripts.js', array( 'jquery' ), '', true );
		wp_register_script( 'hoverall-js', get_stylesheet_directory_uri() . '/HoverAlls/jquery-1.11.1.min.js', array( 'jquery' ), '', true );
		wp_register_script( 'hoverall2-js', get_stylesheet_directory_uri() . '/HoverAlls/jquery-HoverAlls-2.6.min.js', array( 'jquery' ), '', true );

		// enqueue styles and scripts
		wp_enqueue_script( 'bones-modernizr' );
		wp_enqueue_script( 'hoverall-js' );
		wp_enqueue_script( 'hoverall2-js' );
		wp_enqueue_style( 'bones-stylesheet' );
		wp_enqueue_style( 'stylesheet' );
```

The function:
```
jQuery(window).load(function(){
jQuery('.HoverAlls_Caption').HoverAlls({
"mode" : "caption",
"coords" : "0,0||0,0||0,0",
"text_coords" : "0,0||0,0||0,0"
});
});
```

Thanks so much!!!

**Comment by dirad** [comment]
So basically if your files are really in that place - easiest way you can check this is weather you got an error in the console - your code seems fine fore thr register and enqueue.

The hoverall might not be -  and you should check errors for that..

**Comment by jbbow** [comment]
There is definitely an error in the console.  Says they don¹t exist. I don¹t
understand why. I even take the location address and put it in the domain
address box itself and it says it¹s not there.  I can see the files on the
remote server though.

Also, any idea when I upload new css rules they don¹t go into effect for
like 5 minutes or longer?

Thank you so much!!

From:  Adir Dee <notifications@github.com>
Reply-To:  eddiemachado/bones
<reply+00f6ebd9f269bf9d9ce24023a90d5e20c42e0ecf6eafcc4092cf00000001128069a59
2a169ce073ad4aa@reply.github.com>
Date:  Wednesday, December 9, 2015 at 5:13 PM
To:  eddiemachado/bones <bones@noreply.github.com>
Cc:  Jill Bowers <jill@camostudio.com>
Subject:  Re: [bones] CSS styles not showing right away and outside scripts
and stylesheet come up as 404 (#807)


So basically if your files are really in that place - easiest way you can
check this is weather you got an error in the console - your code seems fine
fore thr register and enqueue.

The hoverall might not be -  and you should check errors for that..


Reply to this email directly or view it on GitHub
<https://github.com/eddiemachado/bones/issues/807#issuecomment-163415045> .

**Comment by dirad** [comment]
From what you write i suspect you are working on the wrong directory.
Either that or you are not on the server entirely.

If the full path of the files gives you a 404 page - most likely the file isnt there, or possibly there are wrong permissions for the file.

Try to verify your working environment; Server, directories, which theme folder you are on, weahter you are saving the files to the right place, and provide screenshots or code is best.

It does sound like something in your workflow is misleading you.

**Comment by jbbow** [comment]
Ok, I TOTALLY have a directory issue. I have never experienced this before
so Im not sure how to fix it.  I reinstalled your theme and set it up like I
normally would and still does the same thing. I have did a hard refresh,
cache trash, etcyou name it I did it. I know this isn¹t your problem now as
we know it¹s on my end but if you could give me any advice I¹d appreciate
it.  Permissions are fine.  I was able to find the 2 files that I told you
about by manually putting in the address so I know it¹s a directory.  I
don¹t know even what to give you for you to help me. Sorry. Any advice would
be great!!!!  Thank you!!!

From:  Adir Dee <notifications@github.com>
Reply-To:  eddiemachado/bones
<reply+00f6ebd9f269bf9d9ce24023a90d5e20c42e0ecf6eafcc4092cf00000001128069a59
2a169ce073ad4aa@reply.github.com>
Date:  Wednesday, December 9, 2015 at 5:13 PM
To:  eddiemachado/bones <bones@noreply.github.com>
Cc:  Jill Bowers <jill@camostudio.com>
Subject:  Re: [bones] CSS styles not showing right away and outside scripts
and stylesheet come up as 404 (#807)


So basically if your files are really in that place - easiest way you can
check this is weather you got an error in the console - your code seems fine
fore thr register and enqueue.

The hoverall might not be -  and you should check errors for that..


Reply to this email directly or view it on GitHub
<https://github.com/eddiemachado/bones/issues/807#issuecomment-163415045> .

**Comment by dirad** [comment]
What is the status now?
Do you still get an error? is it still 404 ?
If you found the files, update their uris.



Paste your scripts file.

**Comment by jbbow** [comment]
So, I was able to find the script by typing the full address in.  IT’s all
there and fine.  However, I still have a 15 minute delay to see changes to
the css.  The client I am doing this for has me on a subdomain.  Do you
think that could be the reason?  When I try to set up the ftp to the regular
site and then put the root directory as the sub domain folder, it says I
don’t have permission.  But when I put in the subdomain and treat it like
it’s it’s own site, it works. Am I just reaching now? :)

From:  Adir Dee <notifications@github.com>
Reply-To:  eddiemachado/bones
<reply+00f6ebd94ca965b4e07fb6ae3ef16b2f43e9f5de4d3ccb0b92cf00000001128248529
2a169ce073ad4aa@reply.github.com>
Date:  Friday, December 11, 2015 at 3:16 AM
To:  eddiemachado/bones <bones@noreply.github.com>
Cc:  Jill Bowers <jill@camostudio.com>
Subject:  Re: [bones] CSS styles not showing right away and outside scripts
and stylesheet come up as 404 (#807)

What is the status now?
Do you still get an error? is it still 404 ?
If you found the files, update their uris.



Paste your scripts file.


On Fri, Dec 11, 2015 at 1:47 AM, jbbow <notifications@github.com> wrote:

> Ok, I TOTALLY have a directory issue. I have never experienced this before
> so Im not sure how to fix it. I reinstalled your theme and set it up like I
> normally would and still does the same thing. I have did a hard refresh,
> cache trash, etcŠyou name it I did it. I know this isn¹t your problem now
> as
> we know it¹s on my end but if you could give me any advice I¹d appreciate
> it. Permissions are fine. I was able to find the 2 files that I told you
> about by manually putting in the address so I know it¹s a directory. I
> don¹t know even what to give you for you to help me. Sorry. Any advice
> would
> be great!!!! Thank you!!!
>
> From: Adir Dee <notifications@github.com>
> Reply-To: eddiemachado/bones
>
> <reply+00f6ebd9f269bf9d9ce24023a90d5e20c42e0ecf6eafcc4092cf00000001128069a59
> 2a169ce073ad4aa@reply.github.com>
> Date: Wednesday, December 9, 2015 at 5:13 PM
> To: eddiemachado/bones <bones@noreply.github.com>
> Cc: Jill Bowers <jill@camostudio.com>
> Subject: Re: [bones] CSS styles not showing right away and outside scripts
> and stylesheet come up as 404 (#807)
>
>
> So basically if your files are really in that place - easiest way you can
> check this is weather you got an error in the console - your code seems
> fine
> fore thr register and enqueue.
>
> The hoverall might not be - and you should check errors for that..
>
> ‹
> Reply to this email directly or view it on GitHub
> <https://github.com/eddiemachado/bones/issues/807#issuecomment-163415045>
> .
>
>
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eddiemachado/bones/issues/807#issuecomment-163788631>.
>

—
Reply to this email directly or view it on GitHub
<https://github.com/eddiemachado/bones/issues/807#issuecomment-163871851> .

**Comment by jbbow** [comment]
Ok so I am happy to report that it’s fixed. Apparently when using BlueHost
they have a thing called Varnish and it was enabled. I had to disable it and
now changes show immediately.

Still don’t have an issue for the directory though and why I had to do a
manual link.  Thank you for all your help!! Sorry that it had nothing to do
with your theme.  Love your theme.  Thank you!!

From:  Adir Dee <notifications@github.com>
Reply-To:  eddiemachado/bones
<reply+00f6ebd94ca965b4e07fb6ae3ef16b2f43e9f5de4d3ccb0b92cf00000001128248529
2a169ce073ad4aa@reply.github.com>
Date:  Friday, December 11, 2015 at 3:16 AM
To:  eddiemachado/bones <bones@noreply.github.com>
Cc:  Jill Bowers <jill@camostudio.com>
Subject:  Re: [bones] CSS styles not showing right away and outside scripts
and stylesheet come up as 404 (#807)

What is the status now?
Do you still get an error? is it still 404 ?
If you found the files, update their uris.



Paste your scripts file.


On Fri, Dec 11, 2015 at 1:47 AM, jbbow <notifications@github.com> wrote:

> Ok, I TOTALLY have a directory issue. I have never experienced this before
> so Im not sure how to fix it. I reinstalled your theme and set it up like I
> normally would and still does the same thing. I have did a hard refresh,
> cache trash, etcŠyou name it I did it. I know this isn¹t your problem now
> as
> we know it¹s on my end but if you could give me any advice I¹d appreciate
> it. Permissions are fine. I was able to find the 2 files that I told you
> about by manually putting in the address so I know it¹s a directory. I
> don¹t know even what to give you for you to help me. Sorry. Any advice
> would
> be great!!!! Thank you!!!
>
> From: Adir Dee <notifications@github.com>
> Reply-To: eddiemachado/bones
>
> <reply+00f6ebd9f269bf9d9ce24023a90d5e20c42e0ecf6eafcc4092cf00000001128069a59
> 2a169ce073ad4aa@reply.github.com>
> Date: Wednesday, December 9, 2015 at 5:13 PM
> To: eddiemachado/bones <bones@noreply.github.com>
> Cc: Jill Bowers <jill@camostudio.com>
> Subject: Re: [bones] CSS styles not showing right away and outside scripts
> and stylesheet come up as 404 (#807)
>
>
> So basically if your files are really in that place - easiest way you can
> check this is weather you got an error in the console - your code seems
> fine
> fore thr register and enqueue.
>
> The hoverall might not be - and you should check errors for that..
>
> ‹
> Reply to this email directly or view it on GitHub
> <https://github.com/eddiemachado/bones/issues/807#issuecomment-163415045>
> .
>
>
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eddiemachado/bones/issues/807#issuecomment-163788631>.
>

—
Reply to this email directly or view it on GitHub
<https://github.com/eddiemachado/bones/issues/807#issuecomment-163871851> .

**Comment by dirad** [comment]
not my theme but as a like it just as well at least - i pass the blesses to
@eddiemachado .

cheers

**Comment by Twintails** [comment]
@jbbow please close the issue if you have a moment and it is resolved :-)
